plugins/inputs/sqlserver/sqlserver.go

```diff
@@ -46,62 +47,62 @@ const sampleConfig = `
 ##   See https://github.com/denisenkom/go-mssqldb for detailed connection
 ##   parameters, in particular, tls connections can be created like so:
 ##   "encrypt=true;certificate=<cert>;hostNameInCertificate=<SqlServer host fqdn>"
-# servers = [
-#  "Server=192.168.1.10;Port=1433;User Id=<user>;Password=<pw>;app name=telegraf;log=1;",
-# ]
-
-## This enables a specific set of queries depending on the database type. If specified, it replaces azuredb = true/false and query_version = 2
-## In the config file, the sql server plugin section should be repeated  each with a set of servers for a specific database_type.
-## Possible values for database_type are  
-## "AzureSQLDB" 
-## "SQLServer"
-## "AzureSQLManagedInstance"
+servers = [
+  "Server=192.168.1.10;Port=1433;User Id=<user>;Password=<pw>;app name=telegraf;log=1;",
+]
+
+## "database_type" enables a specific set of queries depending on the database type. If specified, it replaces azuredb = true/false and query_version = 2
+## In the config file, the sql server plugin section should be repeated each with a set of servers for a specific database_type.
+## Possible values for database_type are - "AzureSQLDB" or "AzureSQLManagedInstance" or "SQLServer"
+
+## Queries enabled by default for database_type = "AzureSQLDB" are - 
+## AzureSQLDBResourceStats, AzureSQLDBResourceGovernance, AzureSQLDBWaitStats, AzureSQLDBDatabaseIO, AzureSQLDBServerProperties, 
+## AzureSQLDBOsWaitstats, AzureSQLDBMemoryClerks, AzureSQLDBPerformanceCounters, AzureSQLDBRequests, AzureSQLDBSchedulers
+
 # database_type = "AzureSQLDB"
 
+## A list of queries to include. If not specified, all the above listed queries are used.
+# include_query = []
+
+## A list of queries to explicitly ignore.
+# exclude_query = []
+
+## Queries enabled by default for database_type = "AzureSQLManagedInstance" are - 
+## AzureSQLMIResourceStats, AzureSQLMIResourceGovernance, AzureSQLMIDatabaseIO, AzureSQLMIServerProperties, AzureSQLMIOsWaitstats, 
+## AzureSQLMIMemoryClerks, AzureSQLMIPerformanceCounters, AzureSQLMIRequests, AzureSQLMISchedulers
+
+# database_type = "AzureSQLManagedInstance"
+
+# include_query = []
+
+# exclude_query = []
+
+## Queries enabled by default for database_type = "SQLServer" are - 
+## SQLServerPerformanceCounters, SQLServerWaitStatsCategorized, SQLServerDatabaseIO, SQLServerProperties, SQLServerMemoryClerks, 
+## SQLServerSchedulers, SQLServerRequests, SQLServerVolumeSpace, SQLServerCpu
+
+database_type = "SQLServer"
+
+include_query = []
+
+## SQLServerAvailabilityReplicaStates and SQLServerDatabaseReplicaStates are optional queries and hence excluded here as default
+exclude_query = ["SQLServerAvailabilityReplicaStates", "SQLServerDatabaseReplicaStates"]
+
+## Set this flag to true only when you don't want to use the default server used for collecting SQL Server (database_type = "SQLServer") metrics.
```

Comment:
Worth mentioning if you don't have a connection string defined we will connect to default "local" instance which isn't relevant for Azure.

Reply:
Made the comment more explicit